### PR TITLE
Fix redundancy in CRToastManager regarding showNotification:

### LINKED ISSUE #163: Redundant calls to addNotification:
There are three `showNotification` methods that are meant to be easy-to-use. `showNotificationsWithOptions:completionBlock:` AND `showNotificationWithOptions:appearanceBlock:completionBlock:` call `addNotification` individually. This makes it hard to hook showing notifications, since you can't override addNotification.

### PR DESCRIPTION
**Pull request by jadar**
Resolves #163 

Allows easier hooking of showing notifications.

### CODE REVIEW

**Comment by dmiedema** [comment]
Good catch :+1:

File: CRToast/CRToastManager.m [hunk 1]
```diff
@@ -37,9 +37,9 @@ + (void)setDefaultOptions:(NSDictionary*)defaultOptions {
 }
 
 + (void)showNotificationWithOptions:(NSDictionary*)options completionBlock:(void (^)(void))completion {
```
Comment (dmiedema) [inline CRToast/CRToastManager.m]: Just as a style/organizational thing (thats not your fault but probably mine), I think it'd be prettier/easier to read if this was above `+showNotificationWithOptions:appearanceBlock:completionBlock` but thats just me ¯\\_(ツ)_/¯

Comment (jadar) [inline CRToast/CRToastManager.m]: You mean the definition should be above it? Isn't it that way already?

**Comment by Ashton-W** [comment]
Thanks :star:

**Pull request merged**